Tests/ObfuscationTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ObfuscationTests.swift
+//  UtiliKit-iOSTests
+//
+//  Created by Russell Mirabelli on 7/23/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import XCTest
+@testable import UtiliKit
+
+class ObfuscationTests: XCTestCase {
+
+    func testSimpleObfuscatedString() {
+        let key = ObfuscatedKey().A.B.A.B.value
+        let expected = "ABAB"
+        XCTAssert(key == expected, "Keys do not match: \(key) is not \(expected)")
```

Comment:
Very minor, but you might consider using `XCTAssertEqual()` instead of the more generic `XCTAssert()`.

Reply:
done